### PR TITLE
Show current PR on status bar and allow navigation to PR details MVP

### LINKED ISSUE #1097: Add a margin next to status to show which PR we're on
Add control here so @donokuda can start experimenting with UI

image

### PR DESCRIPTION
**Pull request by jcansdale**
This adds some UI that displays information about any PR associated with the current branch. At the moment it simply shows the PR number and has a tooltop for its title. This was to show what data it is currently populated with.

Maybe we could allow navigation the the PR details pane or let the user toggle modes for adding comments?

This primarily a branch for experimenting with different UI ideas.

image

Fixes #1097

### CODE REVIEW

**Comment by jcansdale** [comment]
@donokuda I'm going to try making this UI do something when clicked on. It will the PR detail pane for the active PR and ideally select the corresponding file in the PR that's currently being edited. We could get even more fancy and navigate to the exact same line (giving the user an easy way to comment).

Please feel free to hack on the UI. I'll work on the wiring up. If I touch the XAML, it's just an example to show the wiring working. Otherwise, the XAML is all yours. Does that make sense? 😄

**Comment by donokuda** [comment]
@jcansdale Yup! I'm currently working on https://github.com/github/VisualStudio/pull/1080 and once that's in a good place I'll hop over in on here.

**Comment by donokuda** [comment]
I tested this branch out but it looks like it's not showing the PR information for me. Is there anything I should do to get it to show? I'm on VS2017 fwiw.

image

**Comment by jcansdale** [comment]
@donokuda Thanks for the heads up.

> I'm on VS2017 fwiw.

Ah, it seems it doesn't work on VS 2017. 😭  At least it degrades gracefully! 😉

**Comment by jcansdale** [comment]
@donokuda,

> I tested this branch out but it looks like it's not showing the PR information for me.

I've added a workaround that will hopefully fix this issue.

The code isn't pretty, but should at least put the UI where it's supposed to be. I'll do it properly when I get the chance. 😄

File: src/GitHub.InlineReviews/InlineReviewsPackage.cs [hunk 1]
```diff
@@ -5,20 +5,31 @@
...
     {
         protected override void Initialize()
         {
             base.Initialize();
             PackageResources.Register(this);
+            ShowPullRequestStatus();
+        }
+
+        void ShowPullRequestStatus()
+        {
```
Comment (jcansdale) [inline src/GitHub.InlineReviews/InlineReviewsPackage.cs]: This should be in its own package, no piggy backing off `InlineReviewsPackage`.

File: src/GitHub.InlineReviews/Views/PullRequestStatusView.xaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+             mc:Ignorable="d" 
+             d:DesignHeight="20" d:DesignWidth="60">
+
+    <Grid Visibility="{Binding Number, TargetNullValue=Collapsed}">
+        <Button Command="{Binding ShowCurrentPullRequestCommand}">
+            <TextBlock>PR #<Run Text="{Binding Number}" /></TextBlock>
+        </Button>
+        <Grid.ToolTip>
+            <TextBlock Text="{Binding Title}" />
+        </Grid.ToolTip>
```
Comment (jcansdale) [inline src/GitHub.InlineReviews/Views/PullRequestStatusView.xaml]: @donokuda here's the UI, if you can call it that. ;-)

**Comment by jcansdale** [comment]
This is now merged with master and should be working again. Not sure why the CI build just failed.

**Comment by jcansdale** [comment]
Starting off the feature branch with an MVP version.

**Pull request merged**